Source/Core/Core/TitleDatabase.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <fstream>
+#include <iostream>
+
+#include "Core/TitleDatabase.h"
+
+#include "Common/FileUtil.h"
+#include "Common/MsgHandler.h"
+#include "Common/StringUtil.h"
+#include "Core/ConfigManager.h"
+#include "DiscIO/Enums.h"
+
+namespace Core
+{
+static std::string GetLanguageCode(DiscIO::Language language)
+{
+  switch (language)
+  {
+  case DiscIO::Language::LANGUAGE_JAPANESE:
+    return "ja";
+  case DiscIO::Language::LANGUAGE_ENGLISH:
+    return "en";
+  case DiscIO::Language::LANGUAGE_GERMAN:
+    return "de";
+  case DiscIO::Language::LANGUAGE_FRENCH:
+    return "fr";
+  case DiscIO::Language::LANGUAGE_SPANISH:
+    return "es";
+  case DiscIO::Language::LANGUAGE_ITALIAN:
+    return "it";
+  case DiscIO::Language::LANGUAGE_DUTCH:
+    return "nl";
+  case DiscIO::Language::LANGUAGE_SIMPLIFIED_CHINESE:
+    return "zh_CN";
+  case DiscIO::Language::LANGUAGE_TRADITIONAL_CHINESE:
+    return "zh_TW";
+  case DiscIO::Language::LANGUAGE_KOREAN:
+    return "ko";
+  default:
+    return "en";
+  }
+}
+
+TitleDatabase::TitleDatabase()
+{
+  // Titles that cannot be part of the Wii TDB,
+  // but common enough to justify having entries for them.
+
+  // i18n: "Wii Menu" (or System Menu) refers to the Wii's main menu,
+  // which is (usually) the first thing users see when a Wii console starts.
+  m_title_map.emplace("0000000100000002", GetStringT("Wii Menu"));
+  m_title_map.emplace("OHBC", "The Homebrew Channel");
```

Comment:
I'm not sure the HBC should be localised, tbh. I don't know about other languages, but in French it's referred to as "the Homebrew Channel" most of the time. Since the artwork (channel banner + in-channel graphics) only says "the Homebrew Channel", I wouldn't be surprised if this was the case in other languages though. If you know this isn't the case, let me know and I'll add the GetStringT.

I'll add all the other IDs of the HBC to the default title map, just in case.